.github/workflows/main.yml

```
@@ -27,28 +27,34 @@ jobs:
+      - name: Build and analyze with SonarCloud
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: |
-          git fetch --unshallow
-          mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.projectKey=ita-social-projects-green-city -Dsonar.organization=ita-social-projects -Dsonar.host.url=https://sonarcloud.io -Dsonar.binaries=target/classes -Dsonar.dynamicAnalysis=reuseReports
+          mvn -B clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar \
+           -Dsonar.projectKey=GreenCity-UA-4146-4625-01_GreenCityMVP21 \
+           -Dsonar.organization=greencity-ua-4146-4625-01 \
+           -Dsonar.host.url=https://sonarcloud.io \
+           -Dsonar.binaries=target/classes 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)